src/GenAPI/Microsoft.DotNet.GenAPI/SyntaxGeneratorExtensions.cs

```diff
@@ -147,4 +148,9 @@ private static SyntaxNode DestructorDeclaration(this SyntaxGenerator syntaxGener
             return decl.ReplaceNode(decl.ReturnType, SyntaxFactory.IdentifierName(SyntaxFactory.Identifier(string.Empty)));
         }
     }
+
+    internal static class SyntaxNodeExtensions
+    {
+        public static SyntaxNode Rewrite(this SyntaxNode node, CSharpSyntaxRewriter rewriter) => rewriter.Visit(node);
+    }
```

Comment:
Should this class be in a separate file given that file name is called `SyntaxGeneratorExtensions.cs`?